sklearn/preprocessing/tests/test_data.py

```diff
@@ -1641,3 +1700,13 @@ def test_fit_cold_start():
         # with a different shape, this may break the scaler unless the internal
         # state is reset
         scaler.fit_transform(X_2d)
+
+
+def test_quantile_normalizer_pickling():
+    qn = QuantileNormalizer()
+    qn.fit(iris.data)
+
+    qn_ser = pickle.dumps(qn, pickle.HIGHEST_PROTOCOL)
+    qn2 = pickle.loads(qn_ser)
+    assert_array_almost_equal(qn.transform(iris.data),
+                              qn2.transform(iris.data))
```

Comment:
You should also check that it can pickle correctly before fitting (evenn though it should trivially work).